### ISSUE TITLE
Title slot for i-tab component

### ISSUE THREAD

**Issue by marbocub** [inkline/inkline#213]
* **I'm submitting a:**
    - [x] Feature Request
  
* **Summary**
    Title slot for i-tab component.

* **Can you provide a basic example?** 
    ~~~
    <i-tabs>
        <i-tab>
            <template v-slot:title>
                <i class="fas fa-exclamation"></i>
                Tab title
            </template>
            Tab content
        </i-tab>
    </i-tabs>
    ~~~

* **Motivation**
    When including an icon in a tab title, it's simpler to use a slot than to use custom headings.

**Comment by alexgrozav** [comment]
Hey! Because the tab headers are generated automatically inside the `<i-tabs>` component, you can provide have a custom header there. 

I'll investigate how to provide a title slot via `i-tab`, but you can use the `i-tabs` `header` scoped slot as a workaround until I find a better way to do it.

**Comment by alexgrozav** [comment]
Thank you for this! I've released 2.3.3 containing your changes.
